std/experimental/logger/package.d

```diff
@@ -1,19 +1,29 @@
 /**
 Implements logging facilities.
 
+Copyright: Copyright Robert "burner" Schadek 2013 --
```

Comment:
Why was the copyright notice moved to the top? Usually in Phobos modules they appear toward the bottom of the module docs (before the module members).